src/xcode/ENA/ENA/Source/Scenes/Home/HomeBadgeWrapper.swift

```diff
@@ -0,0 +1,54 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+final class HomeBadgeWrapper {
+
+	// MARK: - Init
+
+	init(
+		_ values: [Badges: Int?]
+	) {
+		badgesCount = values
+	}
+
+	convenience init() {
+		self.init([:])
+	}
+
+	// MARK: - Internal
+
+	enum Badges: Int, CaseIterable, Hashable {
+		case unseenTests = 0
+		case riskStateIncreased
+	}
+
+	var updateView: ((String?) -> Void)?
+	private var badgeString: String? {
```

Comment:
Can this be renamed?

```suggestion
	private var badgeValue: String? {
```